clang/test/Interpreter/lambda.cpp

```diff
@@ -1,7 +1,8 @@
 // REQUIRES: host-supports-jit
 // UNSUPPORTED: system-aix
 // RUN: cat %s | clang-repl | FileCheck %s
-// RUN: cat %s | clang-repl -Xcc -O2 | FileCheck %s
+// RUN: cat %s | not clang-repl -Xcc -Xclang -Xcc -verify -Xcc -O2 | FileCheck %s
```

Comment:
We should not need a `not` here. You can probably avoid it by adding a correct C++ code before calling `quit`.

Reply:
Sure shall take care of it. Thanks